docs/detections/osquery-response-action.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[osquery-response-action]]
+=== Add Osquery Response Actions
+Osquery Response Actions allow you to add live queries to custom query rules so you can automatically collect data on systems the rule is monitoring. Use this data to support your alert triage and investigation efforts.
+
+NOTE: Response Actions are in technical preview.
+
+[role="screenshot"]
+image::images/available-response-actions.png[Shows available response actions]
+
+.Requirements
```

Comment:
Maybe move the reqs above the screenshot -- they kind of get buried under the image?